src/components/Welcome/Welcome.tsx

```diff
@@ -14,20 +13,14 @@ export function Welcome() {
           inherit
           variant="gradient"
           component="span"
-          gradient={{ from: 'pink', to: 'yellow' }}
+          gradient={{ from: 'blue', to: 'purple' }}
```

Comment:
Why did you changes this? those are the starter files that came out with the template